Bugzilla/Elastic/Indexer.pm

```diff
@@ -213,7 +155,8 @@ sub _select_updated_ids {
 sub bulk_load_ids {
     my ($self, $class, $ids) = @_;
 
-    $self->put_mapping($class);
+    $self->_create_index($class);
```

Comment:
This was unexpected - why did you add a _create_index here? Did it not work previously?